client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import warnings
+
+from verta._protos.public.monitoring import (
+    DataMonitoringService_pb2 as _DataMonitoringService,
+)
+
+from verta._tracking import (entity, _Context)
+from verta._internal_utils import (
+    _utils,
+)
+
+
+class MonitoredEntity(entity._ModelDBEntity):
+    def __init__(self, conn, conf, msg):
+        super(MonitoredEntity, self).__init__(
+            conn, conf, _DataMonitoringService, "monitored_entity", msg
+        )
+
+    def __repr__(self):
```

Comment:
This PR has 3 different formats for its `__repr__()`s 😬. The client also has multiple formats grandfathered in; I think the whole thing is due to be cleaned up to match.

I'm personally a fan of `"\n\t".join()`